spaceship.zsh

```diff
@@ -386,6 +408,7 @@ spaceship_prompt() {
   spaceship_ruby_version
   spaceship_xcode_version
   spaceship_swift_version
+  spaceship_golang_status
```

Comment:
I checked `gvm` and it has no way to check the current `go` version (like `nvm current`). Looks like `go version` is the default way of doing that. So I just renamed it to `spaceship_golang_version`.